app/views/browse/pod.html

```diff
@@ -124,19 +124,60 @@
                     ng-init="containers = pod.status.containerStatuses"
                     ng-if="containersRunning(pod.status.containerStatuses) > 0">
                     <uib-tab-heading>Terminal</uib-tab-heading>
-                    <div>
+
+                    <div class="mar-bottom-md">
                       <span class="pficon pficon-info" aria-hidden="true"></span>
-                      When you navigate away from this pod, any processes running
-                      in these terminals will quit.
+                      Terminal names with <i class="fa fa-terminal btn-success"></i> are active, healthy terminal sessions to living containers.
```

Comment:
I'd just update the sentence here to something like "When you navigate away from this pod any open terminal connections will be closed.  This will kill any foreground processes you started from the terminal."

If you need to designate what the icons mean, do it in a simple legend underneath,  something like
check-icon - Connected terminal
times-icon - Disconnected terminal

could be inline if we need to save some vertical height.  Alternatively as Ben suggested, adding text after the container name would eliminate the need for explaining the icons.